### PR TITLE
August2020 updates

### PR DESCRIPTION
**Pull request by kate-hansen**
Feedback welcome!  Please help me word some of this new stuff.

### CODE REVIEW

File: benefits.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 We offer great benefits so that the **life** portion of work-life balance can be more enjoyable and a little less stressful.
 
-* [Free employee healthcare](#Health-Insurance) ([health](#health), [dental](#dental), and [vision](#vision))
-* [Vacation](#vacation) - 15 days per year
-* [Holidays](#holidays) - 12 per year
-* [Sick](#sick) days as necessary
-* [401k](#401k) - 3% automatic employer contribution
-* [Hardware](#hardware) - you choose
+- [Free employee healthcare](#Health-Insurance) ([health](#health), [dental](#dental), and [vision](#vision))
```
Comment (kate-hansen) [inline benefits.md:5]: This looks like an autochange from my linter, sorry.

File: code-of-conduct.md [hunk 2]
```diff
@@ -3,38 +3,64 @@
 Openness, honesty, and empathy are the building blocks of our culture. Being excellent to each other is the ultimate goal, and we recognize that setting clear guidelines around what constitutes "being excellent" can be helpful and reassuring for all team members. With that, we've put together this Code of Conduct. As with any portion of this handbook, if you feel something is missing from this section, let's work together to update it.
 
 ## Dress Code
+
```
Comment (kate-hansen) [inline code-of-conduct.md:6]: linter stuff again.

File: exiting.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+### Resignation
+
+When leaving Base Two, please provide a signed letter of resignation, or email your supervisor letting them know that you will be leaving your position and when your last day will be. Please give as much notice as possible, as it's the nice thing to do. Your last day cannot be a vacation day, sick day, personal day, or holiday. Please make arrantements to return all Base Two property including laptops and peripherals, office keys, or any other equiptment before your last day.
```
Comment (tborres) [inline exiting.md]: Typos
```suggestion
When leaving Base Two, please provide a signed letter of resignation, or email your supervisor letting them know that you will be leaving your position and when your last day will be. Please give as much notice as possible, as it's the nice thing to do. Your last day cannot be a vacation day, sick day, personal day, or holiday. Please make arrangements to return all Base Two property including laptops and peripherals, office keys, or any other equipment before your last day.
```

File: code-of-conduct.md [hunk 4]
```diff
@@ -3,38 +3,64 @@
...
+## Reasonable and Fair Accomodation for Persons with Disabilities
+
+Base TWo will provide reasonable and fair accomodation to inidividuals with a physical or mental disability. Criteria being that accomodation would not impose an undue hardship on Base Two or other employees, and would enable the individual to apply for employment or perform essential functions of their position. It is the responsiblity of every applicant or employee who requires accomodation to make his or her needs known, and understand that Base Two will engage in this accomodation process in good faith. For accomodations related to completing a job application, please contact [info@base2.io](mailto:jobs@base2.io).
+
+## At Will Employment
+
+This handbook serves as employee guidlines, not as an employment contract.
```
Comment (kate-hansen) [inline code-of-conduct.md]: In doing some handbook research, this was noted and I thought it's worth outlining in writing.

File: who-we-are.md [hunk 5]
```diff
@@ -32,26 +28,45 @@ We promote a company culture that appreciates collaborative professional growth.
...
-
-Career growth is facilitated and tracked via the Buddy System. Individuals meet routinely with a buddy of their choice (not all team members are buddies, but every team member has a buddy) to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a manager, and the Buddy System is no different: we do not review our teammates, we advocate for them. All Buddy System documents are transparent and [available on the shared Drive](https://b2io.slack.com/archives/C904HLD5L/p1516982662000944).
+In addition we also provide access to online training opportunities and sponsor employee attendance at various technical conferences. We encourage community involvement through technical meetups and open source contributions.
 
+Career growth is facilitated and tracked. Individuals meet routinely with their supervisor to discuss career goals, track goal progress, and discuss feedback from the rest of the team. Our flat hierarchy means that no one has a direct manager. we do not review our teammates, We advocate for them.
```
Comment (tborres) [inline who-we-are.md]: Here and elsewhere I think we want to find a different word for "supervisor". It just doesn't feel like the right terminology for how Base Two is structured, but perhaps I'm being overly sensitive.

File: exiting.md (hunk 3 above)
Comment (kate-hansen) [reply exiting.md]: Sorry, working through these now, thanks!

File: who-we-are.md (hunk 5 above)
Comment (kate-hansen) [reply who-we-are.md]: You are faster than I am, I was going to flag that.  I agree, but I couldn't figure out an alternate.  Any ideas?

File: benefits.md (hunk 1 above)
Comment (tborres) [reply benefits.md:5]: No worries. Mine would have done the same thing. Let's stick with the updates.

File: day-to-day.md [hunk 6]
```diff
@@ -177,3 +169,25 @@ Our goal is to be prepared for client meetings 15 minutes before they begin, reg
 ### Remote Meetings
 
 We use [Zoom](https://zoom.us/) for remote meetings. You can sign-up and use Zoom for free with your Base Two email address. The free version works great for 1-1 video calls, but is limited to 40 minutes when there are multiple attendees. We have a paid account through info@base2.io, which is useful for client meetings. The password for the paid account can be found in the [credentials document](https://b2io.slack.com/archives/C904HLD5L/p1516982426000280).
+
+### Base Two Day
+
+Base Two Day is celebrated annually on August 26. It is the anniversary of the founding of Base Two. In the past, this day has been celebrated by hanging out, or participating in Hackathons. Tracking of what has been done on this day is now kept on the team's Google Drive. Discussions around this topic can be found on the [#side-quests](https://join.slack.com/share/zt-h7xz6k8g-FuetGuYYeuKYj34ukKptCg) Slack channel. Base Two Day is also a day when we try to remember to review the handbook!
+
+## We Use
```
Comment (kate-hansen) [inline day-to-day.md:176]: Flagging so that everyone has a chance to add any software / tool recommendations.

File: code-of-conduct.md [hunk 7]
```diff
@@ -3,38 +3,64 @@
...
+
+## Reasonable and Fair Accommodation for Persons with Disabilities
+
+Base TWo will provide reasonable and fair accommodation to individuals with a physical or mental disability. Criteria being that accommodation would not impose an undue hardship on Base Two or other employees, and would enable the individual to apply for employment or perform essential functions of their position. It is the responsibility of every applicant or employee who requires accommodation to make his or her needs known, and understand that Base Two will engage in this accommodation process in good faith. For accommodations related to completing a job application, please contact [info@base2.io](mailto:jobs@base2.io).
```
Comment (tborres) [inline code-of-conduct.md]: Typo fix
```suggestion
Base Two will provide reasonable and fair accommodation to individuals with a physical or mental disability. Criteria being that accommodation would not impose an undue hardship on Base Two or other employees, and would enable the individual to apply for employment or perform essential functions of their position. It is the responsibility of every applicant or employee who requires accommodation to make his or her needs known, and understand that Base Two will engage in this accommodation process in good faith. For accommodations related to completing a job application, please contact [info@base2.io](mailto:jobs@base2.io).
```

File: who-we-are.md [hunk 8]
```diff
@@ -32,26 +28,45 @@ We promote a company culture that appreciates collaborative professional growth.
...
 #### Cost of Living Adjustments
 
 Cost of living adjustments are applied each January based on the previous year's [Social Security COLA](https://www.ssa.gov/oact/cola/colaseries.html). The adjustment applies to all full-time employees as of 12/31 of the previous year.
 
-
 #### Raises
 
 If a team member believes that they (or any other member of the team) deserve a raise, then it should be brought up for discussion either in Buddy meetings, small groups, the Weekly Meeting, or whichever setting feels most comfortable. These discussions have tended to be informal in the past, and focus on the value and responsibilities that the team member provides.
+
+#### Conferences
```
Comment (kate-hansen) [inline who-we-are.md:51]: Add the conferences you've attended while at B2 here! (I went to DrupalGC like a month before I started here, so I'm counting that.)

File: day-to-day.md [hunk 9]
```diff
@@ -4,51 +4,47 @@
...
 
 We use [Harvest](https://b2io.harvestapp.com/) to track time, and there are [nice apps out there for it](http://lmgtfy.com/?q=harvest+apps) to help you track as you work.
 
-Our benchmark number is between 32 and 40 tracked hours, and we operate in good
-faith that we are doing the best we can for the company without burning out.
-
-> Question: at what point do we, as a company, start pestering people about
-> their hours? Should one make up the time spent in a long lunch that wasn't a
-> meeting?
+In order for us to fullfil our obligations to clients and to best work with your team, we ask that you are present at predictable times. Our benchmark number is between 35 and 40 tracked hours, and we operate in good faith that we are doing the best we can for the company without burning out. However, if you are not meeting expectations for reliable time tracking or attendance, another teammate will reach out to you to ask if the issues impeding reliability are fixable. We don't expect you to be logged in from 9-5, but appropriate and reliable communication is paramount when working with a team and for clients.
```
Comment (tborres) [inline day-to-day.md]: This should probably be between 37 and 42 hours tracked.

File: day-to-day.md [hunk 10]
```diff
@@ -4,51 +4,47 @@
...
 
 - Not sure what to track to? Ask! (Seriously.)
 - We round up or down to the nearest quarter hour to streamline our billing process
 - Everyone can see everyone's time.
-  - Don't panic, no one's stalking you. Except maybe Tony.
+  - Don't panic, no one's stalking you (see code of conduct). Except maybe Tony.
```
Comment (tborres) [inline day-to-day.md]: Link to code of conduct

**Review by tborres — COMMENTED** [review]
Throwing a couple comments over the fence. I didn't have enough time to read through it all, but thanks @k8hansen for the updates!

File: day-to-day.md [hunk 11]
```diff
@@ -4,51 +4,47 @@
...
-### What We Expect
+### We Expect
 
 - Your best effort to do what's best for you and the company
-- Be open, honest, and empathetic
-- Clearly communicate with the team
-- Be understanding and constructive in conversations
+- Openness, honesty, and empathy
+- Clear communication with the team
+- Conversations to be both understanding and constructive
```
Comment (sludemann) [inline day-to-day.md]: Maybe?
```suggestion
- Conversations to be both considerate and constructive
```

File: day-to-day.md [hunk 12]
```diff
@@ -4,51 +4,47 @@
...
+### We DON'T Expect
 
-- You know everything on the first day, or ever for that matter
-- You are at your desk at all times
-- You work as many hours as possible
-- Unquestioning agreement with the way things are
+- That you know everything on the first day, or ever for that matter
+- You to be working at your desk at all times
+- You to be working as many hours as possible
+- Your unquestioning agreement with the way things are
```
Comment (sludemann) [inline day-to-day.md]: Maybe?
```suggestion
- Knowing everything
- Being at your desk at all times
- Working as many hours as possible
- Unquestioning agreement with the way things are
```

File: day-to-day.md [hunk 13]
```diff
@@ -4,51 +4,47 @@
 
 Your work and activities may change day-to-day, but there are general expectations that remain the same.
 
-### What We Expect
+### We Expect
 
 - Your best effort to do what's best for you and the company
```
Comment (sludemann) [inline day-to-day.md]: The phrasing of this now feels out of place with the rest of the items.  Maybe something like:
- Do the best you can
- Self care

File: day-to-day.md (hunk 9 above)
Comment (sludemann) [reply day-to-day.md]: That feels a little more arbitrary to me and I think it's worth having an "expected" cap at 40 because _ideally_ we shouldn't be having to do over time to meet our expectations. Obviously reality get's in the way but still.

File: on-boarding.md [hunk 14]
```diff
@@ -8,23 +8,21 @@ One team member will be in charge of supervising the on-boarding process, making
...
 - Add yourself to the [Base Two website](http://base2.io)
-	- Go to [Flow](https://b2io.slack.com/archives/C904HLD5L/p1516982736000566)
-	- Navigate to the base2.io board
-	- Add a task such as "add <name> to the website"
-	- Begin working on the task and track time to the base2.io Harvest project
-- Get to know each member of the team (lunch is a great opportunity for this)
-- Meet your teammates via the pre-scheduled, informal 1:1 meetings that are on your calendar
-	- If you need to reschedule any meeting for any reason, please feel free to do so
+  - Open up the [project board](https://github.com/b2io/base2.io/projects) for the Base 2 website
+  - Add a task such as "add <name> to the website"
```
Comment (sludemann) [inline on-boarding.md]: 🕵️ Casing?
```suggestion
  - Add a task such as "Add <name> to the website"
```

File: who-we-are.md (hunk 5 above)
Comment (sludemann) [inline who-we-are.md]: It was Drew as part of the review process the last time, but conceivably it could be some other person at some point so just simplify to `team member`?
```suggestion
Career growth is facilitated and tracked. Individuals meet routinely with a team member to discuss career goals, track goal progress, and discuss feedback. Our flat hierarchy means that no one has a direct manager. We are accountable to, and advocate for each other.
```

**Review by sludemann — COMMENTED** [review]
Left some feedback, thanks for tackling this!

File: benefits.md [hunk 15]
```diff
@@ -19,65 +19,65 @@ Health insurance is provided through Anthem Blue Cross Blue Shield. Detailed inf
...
-* Up-to 40% off additional materials
+- Exam copay: \$10
+- Prescription lens copay: \$20
+- Exam once every 12 months
+- Lens once every 12 months
+- Up-to 40% off additional materials
 
 ## Vacation
 
 Employees are encouraged to take an average of 3 weeks (15 days) of vacation a year in order to take a break from work, travel, and come back feeling refreshed. Since nobody likes vacation day math, our vacation policy is flexible - there is no loss of vacation days or rollovers at the end of the year. With the flexible policy, if you took a couple of long weekends earlier in the year, and then you have the opportunity to take a 3 week European vacation - do it! We don't want you to miss out on opportunities because of an arbitrary number of days.
```
Comment (reeddunkle) [inline benefits.md]: 💭 Remove the last part of the last sentence? "Arbitrary" feels inaccurate, and in general that part of the sentence feels unnecessary.
```suggestion
Employees are encouraged to take an average of 3 weeks (15 days) of vacation a year in order to take a break from work, travel, and come back feeling refreshed. Since nobody likes vacation day math, our vacation policy is flexible - there is no loss of vacation days or rollovers at the end of the year. With the flexible policy, if you took a couple of long weekends earlier in the year, and then you have the opportunity to take a 3 week European vacation - do it! We don't want you to miss out on opportunities.
```

File: benefits.md [hunk 16]
```diff
@@ -19,65 +19,65 @@ Health insurance is provided through Anthem Blue Cross Blue Shield. Detailed inf
...
 
-When planning to take vacation time we ask that you include your anticipated out of office days to your Base Two calendar, and let the team and your active clients know of your plans. Also, as your time off approaches please remind both the Base Two team and your active clients so that nobody is surprised when they see pictures of you sipping mai tais.
+When planning to take vacation time we ask that you include your anticipated out of office days to Forecast (part of Harvest), as well as to your Base Two calendar, and let the team and your active clients know of your plans. Also, as your time off approaches please remind both the Base Two team and your active clients.
```
Comment (reeddunkle) [inline benefits.md]: 💭 Hyperlink to Forecast? The link I use is `https://forecastapp.com/715825/schedule/team`.

File: business-development.md [hunk 17]
```diff
@@ -4,32 +4,28 @@ Our team is comprised entirely of developers, which makes the business developme
...
-
+We are not a sales-oriented company. We're not out there every day trying to find the next gig. In order to stay in business, our strategy is to develop _sustainable_ relationships with our clients. We make sure we're always providing value to our clients since we don't have a huge pool of leads. We're flexible at the beginning of a relationship as we figure out how best to work together with our clients, and we ask our clients to be flexible through the evolving process.
 
 ### Who We Work With
 
-Our roster runs the gamut from small, bootstrap startups to fortune 500 companies and everything in between. It may sound crazy for such a small team but we have 3-4 billion-dollar clients that we work with, and we also have 3-4 shoestring clients we work with. We're flexible, and we're careful with our clients' money.
-
+Our roster ranges from small, bootstrap startups to fortune 500 companies and everything in between. It may sound crazy for such a small team but we have 3-4 billion-dollar clients that we work with, and we also have 3-4 shoestring clients we work with. We're flexible, and we're careful with our clients' money.
```
Comment (reeddunkle) [inline business-development.md]: 💭 Possible slim down

```suggestion
Our roster ranges from multibillion-dollar, fortune 500 companies to small startups. We're flexible, and we're careful with our clients' money.
```

File: business-development.md [hunk 18]
```diff
@@ -4,32 +4,28 @@ Our team is comprised entirely of developers, which makes the business developme
...
 
 ## Conversation End Goal
 
-Flip the conversation (get on the "same side of the table") to collaboratively solve a problem. Even if we're not the right return-on-the-dollar for them, how can we be helpful? Can we recommend a contractor, can we recommend other companies?
+We strive to collaboratively solve each problem with our client. Even if we're not the right return-on-the-dollar for them, how can we be helpful? Can we recommend a contractor, can we recommend other companies?
```
Comment (reeddunkle) [inline business-development.md]: The notion of "getting on the same side of the table" as them has come up as an explicit conversational strategy in the past. I'm not sure that "strive to collaboratively solve" is as useful as an end-goal strategy. Maybe we should reword the original idea to be more direct. This is probably too wordy, so anyone please help me pare it down:

> We're most successful with clients when we're able to transition from being seen as a vendor trying to sell a solution, to instead them feeling that we're hearing and understanding their problem, brainstorming possible solutions. We think of this as trying to get on the same side of the table as the client, collaborating on their issue. We want them to feel that we're good listeners for any problems they might have, not only those that they believe are explicitly in our wheelhouse. The contracts and sales tend to flow naturally out of a relationship where they first and foremost feel comfortable brainstorming or just venting about whatever problems might be on their plate.

File: code-of-conduct.md [hunk 19]
```diff
@@ -3,38 +3,64 @@
...
 
 In any instance where a team member comes to you with an issue, first ask the team member if they would like you to take action. If the answer is yes:
+
 - Commit to assisting in resolving their issue, or identify a person who can help them
 - Channel your childhood [peer mediation techniques](http://www.sese.org/wp-content/uploads/2013/08/Peer-Mediation-guide.pdf)
 - Act as an unbiased third party
 - Strive for confidentiality as much as possible
 
-We don't have much of a formal system yet, it's something we are continuing to create and refine as a team. If you have ideas, do not hesitate to share them.
+We don't have much of a formal system, it's something we are continuing to create and refine as a team. If you have ideas, do not hesitate to share them.
```
Comment (reeddunkle) [inline code-of-conduct.md:36]: 💭 Possible re-word

```suggestion
Because we don't have a formal system, we're often refining this process as a team. If you have ideas, please share them.
```

File: code-of-conduct.md [hunk 20]
```diff
@@ -3,38 +3,64 @@
...
+
+## Reasonable and Fair Accommodation for Persons with Disabilities
+
+Base TWo will provide reasonable and fair accommodation to individuals with a physical or mental disability. Criteria being that accommodation would not impose an undue hardship on Base Two or other employees, and would enable the individual to apply for employment or perform essential functions of their position. It is the responsibility of every applicant or employee who requires accommodation to make his or her needs known, and understand that Base Two will engage in this accommodation process in good faith. For accommodations related to completing a job application, please contact [info@base2.io](mailto:jobs@base2.io).
+
+## At Will Employment
+
+This handbook serves as employee guideline, not as an employment contract.
+
+> should we include an At-Will statement here?
```
Comment (reeddunkle) [inline code-of-conduct.md:66]: I'm not sure. Has anyone seen an example that they like?

File: day-to-day.md (hunk 10 above)
Comment (reeddunkle) [reply day-to-day.md]: Should we remove the Tony joke? I can read it both as harmless and as an unnecessary tease.

Also, technically the comma should be a semi-colon or a period.

```
Don't panic. No one's stalking you.
```

File: day-to-day.md [hunk 21]
```diff
@@ -4,51 +4,47 @@
...
 
 Try to have the day's hours in and rounded to the quarter hour by the end of the day. If we feel like we have a lot of billable work but aren't tracking a high number of billable hours, that
 will likely trigger a conversation around team productivity.
 
 ### Flex time
 
-Clear time tracking allows for a flexible work schedule without feeling guilty. This allows you to mold your work hours to better fit your personal schedule. For example, if you need to get a haircut, and the best time to schedule that is 2pm on a Thursday, you can make up the hour you miss on
-another day (prior or subsequent).
+A clear system for time tracking allows for a flexible work schedule without feeling guilty. This allows you to mold your work hours to better fit your personal schedule. For example, if you need to get a haircut, and the best time to schedule that is 2pm on a Thursday, you can make up the hour you miss on
+another day (prior or subsequent). Communicating your away from keyboard time to your teammates is a nice thing to do.
```
Comment (reeddunkle) [inline day-to-day.md]: 💭 Possible reword. Doesn't have to be this exact wording, but I think "Communicating your away from keyboard time" can be improved.

```suggestion
another day (prior or subsequent). We appreciate you giving teammates a heads up when you'll be out of pocket.
```

File: day-to-day.md [hunk 22]
```diff
@@ -97,11 +93,7 @@ We have also invested in highly-adjustable, ergonomic Steelcase Gesture chairs f
 
 We take coffee pretty seriously (some of us more than others). Coffee beans and other accoutrement are supplied by the company. Feel free to drink any coffee that has been made, and to make more whenever your caffeine-deprived heart desires.
 
-Aside from coffee, we are also serious about stocking water, fizzy water, pop, and beer.
-
-You can track time while drinking, unless you're drinking margaritas with Jeff - that's definitely not billable.
-
-Grocery shopping is typically the responsibility of one person (Jeff). Let that person know what you'd like to have stocked in the fridge.
+Aside from coffee, we are also serious about stocking water, fizzy water, soda, and beer at the office. For remote employees, you should be drinking your own water, fizzy water, soda, and beer.
```
Comment (reeddunkle) [inline day-to-day.md]: 💭 We've had different remote policies in the past, and as far as I know we haven't had much internal discussion about changing it. I think it makes sense to chat about it some more now that we're primarily remote due to Covid and the team growing.

```suggestion
Aside from coffee, we like to stock water, fizzy water, soda, and beer at the office. Currently, we don't reimburse non-coffee treats for remote employees, but we'd like to discuss this further as more of the team has shifted to working remotely.
```

**Review by reeddunkle — COMMENTED** [review]
Thanks for working on this everyone. It's looking better than it ever has.

I didn't make it through all of the files yet but wanted to submit my thoughts so far.

File: day-to-day.md (hunk 10 above)
Comment (kate-hansen) [reply day-to-day.md]: Yeah, I debated this too.  I'm removing it.

File: day-to-day.md (hunk 12 above)
Comment (kate-hansen) [reply day-to-day.md]: Great suggestions!  Thank you, Sean!!

I think I we should keep the "that you know everything on your first day", it made me feel a who lot better to see this.  I also think that it's a kind way of acknowledging that we're all human.  Maybe verbose, but I think that helps get the point across.  I'll clean it up a bit.

File: business-development.md (hunk 18 above)
Comment (kate-hansen) [reply business-development.md]: I think this is perfect, thank you!!

File: day-to-day.md (hunk 21 above)
Comment (kate-hansen) [reply day-to-day.md]: I pushed this a little more, let me know what you think.
